alignak/objects/schedulingitem.py

```diff
@@ -2197,13 +2197,15 @@ def scatter_notification(self, notif, contacts, notifways, timeperiods, macromod
 
         for contact_id in notif_contacts:
             contact = contacts[contact_id]
-            # We do not want to notify again a contact with
-            # notification interval == 0 that has been already
-            # notified. Can happen when a service exit a downtime
-            # and still in critical/warning (and not acknowledge)
+            # We do not want to notify again a contact with notification interval == 0
+            # if has been already notified except if the item hard state changed!
+            # This can happen when a service exits a downtime and it is still in
+            # critical/warning (and not acknowledge)
             if notif.type == "PROBLEM" and \
                     self.notification_interval == 0 \
+                    and self.state_type == 'HARD' and self.state != self.last_hard_state \
```

Comment:
I have a doubt for `elf.state != self.last_hard_state` because if you want a notification when state go fron WARNING to CRITICAL, here it validate the if and so will not send the notification